src/main/resources/application.properties

```diff
@@ -5,15 +5,15 @@ spring.jackson.default-property-inclusion=non_null
 # Example if build.version=1.2.3-internal becomes Version 0.0.18 (1.2.3-internal)
 build.version=
 
-server.port=8080
+server.port=8089
 #server.ssl.key-store: <keystore.p12>
 #server.ssl.key-store-password: <mypassword>
 #server.ssl.keyStoreType: <PKCS12>
 #server.ssl.keyAlias: <tomcat>
 
 ######## EI Front-End
 ei.frontendServiceHost=localhost
-ei.frontendServicePort=8080
+ei.frontendServicePort=8090
```

Comment:
ei.frontendServicePort and server.port should be the same port.

Reply:
done